### ISSUE TITLE
nvme_drv: build errors with gcc 12

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#4845]
When building `drivers/nvme` with gcc 12, the following error occurs:

```
/.../repos/os/src/drivers/nvme/main.cc:802:9: error: reference to 'Mmio' is ambiguous
  802 |         Mmio::Delayer        &_delayer;
      |         ^~~~
In file included from /.../repos/os/include/os/attached_mmio.h:19,
                 from /.../repos/os/src/drivers/nvme/main.cc:26:
/.../repos/base/include/util/mmio.h:77:1: note: candidates are: 'struct Genode::Mmio Genode::Mmio::Mmio'
   77 | {
      | ^
In file included from /.../repos/os/src/drivers/nvme/main.cc:29:
/.../repos/os/include/platform_session/device.h:90:1: note:                 'class Platform::Device::Mmio Platform::Device::Mmio::Mmio'
   90 | {
      | ^
/.../repos/os/include/platform_session/device.h:89:25: note:                 'class Platform::Device::Mmio'
   89 | class Platform::Device::Mmio : Range, Attached_dataspace, public Genode::Mmio
```

and more errors of the same type.

**Comment by cproc** [comment]
Fixed by a50c2f2.

**Comment by cproc** [comment]
Fixup commit 4d18781 uses `using Mmio = Genode::Mmio` instead of the previous changes.
